sale_order_line_packaging_qty/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Sale Order Line Packaging Quantity",
+    "summary": "Define quantities according to product packaging on sale order lines",
+    "version": "14.0.0.1.0",
```

Comment:
```suggestion
    "version": "14.0.1.0.0",
```

Reply:
Thanks for your suggestion. It is updated.